src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -63,69 +61,69 @@ public void setGuiSettings(GuiSettings guiSettings) {
         requireNonNull(guiSettings);
         userPrefs.setGuiSettings(guiSettings);
     }
-
+    //Done
```

Comment:
What's this comment for? Should it be deleted?

Reply:
Not sure how it end up there